### PR TITLE
Updates November

### PR DESCRIPTION
**Pull request by linkfang**
Pulled from https://github.com/EclipseFdn/openadx.eclipse.org/pull/153 and fixed the conflicts

### CODE REVIEW

**Comment by linkfang** [comment]
Hi @ariexi , can you please check this PR to see if all your changes are still there and correct?

Please note, some of your changes are not listed in the "changes" tab, because there might be other PRs containing the same changes had been merged into master before yours. So, if you could also check the changes via the preview site, that would be great! (And it's more accurate on what has been added/updated so far based on this PR)

For example, there is a change related to "OpenADx Big Picture" in your original PR, but it's not listed here. 
However, if you go to https://deploy-preview-158--eclipsefdn-openadx.netlify.app/vision/#openadx-big-picture, you will see it's already been added. (And actually, it's already live on prod if you check the vision page)

Let me know if you have any questions, thanks!

**Comment by ariexi** [comment]
Hi @linkfang, I checked the preview and looks really good!
Thank you so much for your effort!
Best,
Andy

File: content/resources/index.md [hunk 1]
```diff
@@ -25,6 +25,13 @@ Watch the video on OpenADx here on YouTube: https://youtu.be/7gHLkSzWNKA
 - [Vorteile von Open Source fuer ADAS und autonomes Fahren (only in German available)](https://www.elektroniknet.de/elektronik-automotive/assistenzsysteme/vorteile-von-open-source-fuer-adas-und-autonomes-fahren-154085.html)
 - [Open Source Software Drives Indy Autonomous Challenge Racecars](https://newsroom.eclipse.org/eclipse-newsletter/2021/june/open-source-software-drives-indy-autonomous-challenge-racecars)
 - [Q&A: It’s Time to Take a Closer Look at Eclipse iceoryx](https://newsroom.eclipse.org/eclipse-newsletter/2021/june/qa-it%E2%80%99s-time-take-closer-look-eclipse-iceoryx)
+- [Collaborating with Ecosystem Partners to Accelerate “Autonomous Driving for All” by ADLON](https://www.adlinktech.com/en/Connected-Autonomous-Vehicle-Solutions)
```
Comment (ariexi) [inline content/resources/index.md:28]: If possible, please change "ADLON" to "ADLINK"
We can do the change also next time.

**Review by ariexi — APPROVED** [review]

File: content/resources/index.md (hunk 1 above)
Comment (chrisguindon) [reply content/resources/index.md]: I made the update before merging!

**Pull request merged**